fe/.eslintrc.js

```diff
@@ -26,10 +33,38 @@ module.exports = {
     // Ignore `import` statements with Webpack `@` alias in path
     'import/no-unresolved': ['error', { ignore: ['^@'] }],
 
+    // Set maximum line length to 120 columns if line doesn't contain is not url, string, template literal or regex
+    'max-len': ['error', {
+      'code': 120,
+      'ignoreUrls': true,
+      'ignoreStrings': true,
+      'ignoreTemplateLiterals': true,
+      'ignoreRegExpLiterals': true
+    }],
+
     // Do not permit `console` references or `debugger` statements in production bundle
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
+    // Quotes should be single if there is no template literals or quote to be escaped
```

Comment:
```suggestion
    // Requires the use of single quotes wherever possible. Double quotes are allowed so long as the string contains
    // a single quote that would have to be escaped otherwise.
```

Reply:
>_as_ long as

Reply:
[Doesn't really matter](https://dictionary.cambridge.org/grammar/british-grammar/as-and-as-expressions/as-long-as-and-so-long-as), but change it if you want to.